lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -813,6 +818,64 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
             , expectListField 2
                 (#metrics . #nonMyopicMemberRewards) (`shouldBe` Quantity 0)
             ]
+
+    it "STAKE_POOLS_GARBAGE_COLLECTION_01 - \
+        \retired pools are garbage collected on schedule and not before" $
+        \ctx -> do
+            -- The epoch for which we expect to see a pool retirement event.
+            -- See 'testPoolConfigs' for the original source of this value.
+            let expectedRetirementEpoch = 3
+
+            -- The last epoch for which we expect to see a garbage collection
+            -- event. This is an arbitrary point in time that lies beyond the
+            -- expected retirement epoch.
```

Comment:
```suggestion
            -- The last epoch for which we expect to see effective garbage collection
            -- event is epoch 5, since the stake pool retires in epoch 3. 
            -- This is an arbitrary point in time that lies beyond the
            -- expected garbage collection epoch.
```
?

Reply:
Fixed in 684d8a55ff4dc8b1df548edc2252155a94a59cfb.